packages/google-analytics/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { sendPageview, sendEvent, afterCSR } from "@frontity/analytics";
+import GoogleAnalytics from "../types";
+import Root from "./components";
+
+export const getTrackerName = (id: string) =>
+  `tracker_${id.replace(/-/g, "_")}`;
+
+const googleAnalytics: GoogleAnalytics = {
+  roots: { ga: Root },
+  actions: {
+    analytics: {
+      sendPageview,
+      sendEvent,
+      afterCSR
+    },
+    ga: {
```

Comment:
I'm not sure if the namespaces should be `googleAnalytics`, `gtmAnalytics`, `mixpanelAnalytics`, `amplitudeAnalytics` and so on... thoughts?

Reply:
Using `ga` as the namespace didn't convince me. I prefer your idea. 👍